lib/orangetheses.rb

```diff
@@ -1,12 +1,21 @@
 require 'orangetheses/version'
 
 module Orangetheses
+  #$test = true
+  # OAI
   SET = 'com_88435_dsp019c67wm88m'
   PMH_SERVER = 'http://dataspace.princeton.edu/oai/request'
-  PMH_SERVER = 'http://asdspace300l.princeton.edu/oai/request' if $test
+  PMH_SERVER = 'http://updatespace.princeton.edu/oai/request' if $test
   METADATA_PREFIX = 'oai_dc'
 
+  # REST service
+  COMMUNITY_ID = '267'
```

Comment:
As I said in my email, I'd rather see us rely on the handle for the collection than a random database ID (don't know if they change when the migrate, but may as well do the right thing).